src/TourManager/TourManager.Api/Startup.cs

```diff
@@ -22,31 +21,24 @@ public Startup(IConfiguration configuration)
 
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddDbContext<TourDbContext>(
-                options => options.UseSqlServer(
-                        Configuration.GetConnectionString("TourDbContext"),
-                        x => x.MigrationsAssembly("TourManager.Repository.EfCore.MsSql")));
-
-            // auto mapper Configurations
-            var mapperConfig = new MapperConfiguration(mc =>
-            {
-                mc.AddProfile(new MappingConfig());
-            });
-
-            // add mapper configs to services
-            services.AddSingleton(mapperConfig.CreateMapper());
+        {
+            services
+                .AddCors()
+                .AddControllers()
+                .AddFluentValidation(opt => opt.RegisterValidatorsFromAssembly(Assembly.GetExecutingAssembly()))
+                .AddNewtonsoftJson(opt => opt.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore);

```

Comment:
Do we have ReferenceLoops in our project , why we have switched from `System.Text.Json` to `Newtonsoft.Json` ?